templates/volunteer.html

```diff
@@ -20,9 +20,9 @@ <h1>Volunteer for the Conference</h1>
         <div class="content-wrapper">
             <div class="content">
                 <div class="col">
-                    <p><span style="font-weight: 700;">2024-08-30 Update</span>: we cannot accept more volunteers for
+                    <p><span style="font-weight: 700;">2024-08-30 Update</span>: we cannot give out any more volunteers tickets for
                         this year! Thanks to everyone who signed
-                        up, and we're looking forward to expand the team next year.</p>
+                        up. If you already have a ticket and still want to help, reach out to us</p>
```

Comment:
reach out where/how?

Reply:
Via the methods mentioned in the footer. Currently mail, mastodon, linkedin.

Reply:
I think it would have been well to mention that explicitly... people are dumb and blind.